modules/text2vec-contextionary/client/contextionary.go

```diff
@@ -262,6 +286,9 @@ func (c *Client) NearestWordsByVector(ctx context.Context, vector []float32, n i
 		Vector: vectorToProto(vector),
 	})
 	if err != nil {
+		if strings.Contains(fmt.Sprintf("%v", err), "") {
```

Comment:
could you add a small explanation what you're checking for? The error message is empty if you cannot connect to the module?
Also, wouldn't `err.error()` work instead of `mt.Sprintf("%v", err)`?

Reply:
thanks, I forgot to paste the message in